packages/firebase_performance/firebase_performance_platform_interface/lib/src/method_channel/method_channel_http_metric.dart

```diff
@@ -8,14 +8,11 @@ import 'utils/exception.dart';
 
 class MethodChannelHttpMetric extends HttpMetricPlatform {
   MethodChannelHttpMetric(
-    this._methodChannelHandle,
-    this._httpMetricHandle,
     this._url,
     this._httpMethod,
   ) : super();
 
-  final int _methodChannelHandle;
-  final int _httpMetricHandle;
+  late final int _httpMetricHandle;
```

Comment:
Could this not be simplified?  Instead use `int? _httpMetricHandle` within the start/stop methods to detect whether it's been started/stopped already?

Reply:
I've updated implementation, but you still need an additional `boolean` to check if it has ran once. Otherwise, you would set to `_httpMetricHandle ` to `null` again and you would be able to start/stop again.